Gosu/Image.hpp

```diff
@@ -8,6 +8,8 @@
 #include <Gosu/GraphicsBase.hpp>
 #include <memory>
 #include <vector>
+#include <string>
+#include <iostream>
```

Comment:
```suggestion
#include <iostream>
#include <memory>
#include <string>
#include <vector>
```
I try to keep things alphabetical to avoid duplicates in longer lists

Reply:
But, why iostream?

Reply:
Removed iostream, I thought it was, for some odd reason, required to fix a `missing type specifier - int assumed.` error but after cleaning and rebuilding with out it, it works... 🤷🏼